packages/sdk/src/types/TextStyleTypes.ts

```diff
@@ -61,49 +44,42 @@ export type TextStyleUpdateType = {
 
 export interface AppearanceProperties {
     blendMode?: string;
+    dropShadowColor?: string;
+    fillColor?: string;
     opacity?: string;
+    strokeColor?: string;
+    useDropShadow?: boolean;
     useFill?: boolean;
     useStroke?: boolean;
-    useDropShadow?: boolean;
-    fillColor?: string;
-    strokeColor?: string;
-    dropShadowColor?: string;
 }
 
 export interface SelectedTextStyle {
```

Comment:
As written above I prefer the idea of having a common `TextStyle` or `ITextStyle` interface instead of having `CharacterStyle` & `ParagraphStyle` inheriting from `SelectedTextStyle` which has an opiniated name.

So having `CharacterStyle` `ParagraphStyle` & `SelectedTextStyle` implementing one common `SomethingStyle` interface.